packages/atomic/scripts/watch.mjs

```diff
@@ -21,7 +21,7 @@ function checkPort(port) {
 }
 
 async function rebuild(event, fileName) {
-  if (fileName.contains('/e2e/') || fileName.contains('/.e2e.')) {
+  if (fileName?.contains?.('/e2e/') || fileName?.contains?.('/.e2e.')) {
```

Comment:
side fix, the first call to rebuild fail because we do not send filename.
This fixes that